### PR TITLE
fix #23: start using helmet

### LINKED ISSUE #23: Harden Express using something like Helmet?
And since we're using Express, not sure if we want to consider something like [**helmet**](https://www.npmjs.com/package/helmet) to lock down a few settings.

### PR DESCRIPTION
**Pull request by groovecoder**

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -18,6 +19,19 @@ const UserRoutes = require("./routes/user");
...
+  directives: {
+    baseUri: ["'none'"],
+    defaultSrc: ["'none'"],
+    fontSrc: ["fonts.gstatic.com"],
+    frameAncestors: ["'none'"],
+    imgSrc: ["'self'"],
+    scriptSrc: ["'self'"],
+    styleSrc: ["'self'", "fonts.googleapis.com"],
+    reportUri: "/__cspreport__",
+  }
```
Comment (pdehaan) [inline server.js]:
```
/home/travis/build/mozilla/blurts-server/server.js
  33:4  error  Missing trailing comma  comma-dangle
```

File: app-constants.js [hunk 2]
```diff
@@ -23,8 +23,6 @@ const kEnvironmentVariables = [
   "HIBP_STUB_API_ROOT",
   "HIBP_API_ROOT",
   "HIBP_API_TOKEN",
-  "AWS_ACCESS_KEY_ID",
-  "AWS_SECRET_ACCESS_KEY",
```
Comment (pdehaan) [inline app-constants.js]: Were these just orphaned or something?

Comment (groovecoder) [reply app-constants.js]: Yeah, we don't need them anymore. I removed them from my local `.env` and this keeps throwing errors now. I'll remove these from the heroku environment too as we don't upload anything to AWS anymore.

**Comment by groovecoder** [comment]
I'm adding in some more items from the checklist into this PR too.

**Comment by pdehaan** [comment]
Superficially, I think this looks 👌.
Once this lands and gets deployed to Heroku, we can run it through https://observatory.mozilla.org/ again and cross off the relevant items from #55.

File: server.js [hunk 3]
```diff
@@ -18,6 +19,19 @@ const UserRoutes = require("./routes/user");
...
+app.use(helmet.contentSecurityPolicy({
+  directives: {
+    baseUri: ["'none'"],
+    defaultSrc: ["'none'"],
+    fontSrc: ["fonts.gstatic.com"],
+    frameAncestors: ["'none'"],
+    imgSrc: ["'self'"],
+    scriptSrc: ["'self'"],
+    styleSrc: ["'self'", "fonts.googleapis.com"],
+    reportUri: "/__cspreport__",
```
[the remark this answers is not in the document]
Comment (groovecoder) [reply server.js]: I'm going to ask in tomorrow's RRA if this is an endpoint that ops sets up for us, or if we have to implement it.

File: scripts/load-breaches.js [hunk 4]
```diff
@@ -28,6 +32,8 @@ async function handleBreachesResponse(response) {
     const breachesJSON = JSON.parse(response.body);
 
     for (const breach of breachesJSON) {
+      // purify the description going into the DB
+      breach.Description = DOMPurify.sanitize(breach.Description);
```
[the remark this answers is not in the document]
Comment (groovecoder) [reply scripts/load-breaches.js]: Oh, does it allow all tags by default?

**Comment by groovecoder** [comment]
Rebased to make this branch pass Circle.

**Pull request merged**